Makefile

```diff
@@ -72,6 +72,18 @@ generate_credentials:
 create_state_buckets:
 	./scripts/create-state-buckets.sh $(openshift_nameplate) "ggl-cas-storage"
 
+.PHONY: perf_test
+perf_test: ## run performance tests with k6
+perf_test: APP_HOST=http://localhost:3000
+perf_test: APP_API_ROUTE=/api
+perf_test: APP_ROUTE=$(APP_HOST)$(APP_API_ROUTE)
+perf_test: SERVER_HOST=http://127.0.0.1:8000
+perf_test: SERVER_API_ROUTE=/api/registration
+perf_test: SERVER_ROUTE=$(SERVER_HOST)$(SERVER_API_ROUTE)
+perf_test:
+	@k6 -e NUM_APPLICATIONS=100 -e APP_HOST=$(APP_ROUTE) -e SERVER_HOST=$(SERVER_ROUTE) run client/tests/performance/script.js --out csv=k6_results/test_results.csv
```

Comment:
Nit: Looks like NUM_APPLICATIONS is unused & is just leftover from CCBC's implementation. I'll remove it in my next PR.